tests/UI/pages/BO/orders/view.js

```diff
@@ -637,6 +671,71 @@ class Order extends BOBasePage {
     return this.getPriceFromText(page, this.orderTotalDiscountsSpan);
   }
 
+  /**
+   * Create new invoice
+   * @param page {Page} Browser tab
+   * @returns {Promise<void>}
+   */
+  async createNewInvoice(page) {
+    await this.selectByVisibleText(page, this.addProductInvoiceSelect, 'Create a new invoice');
+  }
```

Comment:
Can we create a more generic function to select from the option with the option by default is `'Create a new invoice'`